pyccel/ast/numpyext.py

```diff
@@ -624,6 +624,23 @@ def __init__(self, start, stop, num=None, endpoint=True, dtype=None):
             else:
                 raise TypeError('cannot determine the type of {}'.format(self))
 
+        args      = (start, stop)
+        integers  = [e for e in args if e.dtype is NativeInteger()]
+        reals     = [e for e in args if e.dtype is NativeReal()]
```

Comment:
See PR #1012

Reply:
Would you mind also renaming the variable. I already did all the others in pyccel so we stop getting confused between real (not imaginary) and "real" (float)

Reply:
yes i already done it

Reply:
Great. Thanks :)